### PR TITLE
Add search to workoutsession

### LINKED ISSUE #72: As a user, I would like to search for a particular type of excercise
so I can see how many sets I have done already

### PR DESCRIPTION
**Pull request by yujinyang1998**

### CODE REVIEW

**Comment by CFZeon** [comment]
LGTM!

**Review by CFZeon — APPROVED** [review]

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java [hunk 1]
```diff
@@ -2,15 +2,34 @@
...
+import workout.workoutsession.WorkoutSessionParser;
 import workout.workoutsession.exercise.Exercise;
 
 import java.util.ArrayList;
 
 public class WorkoutSessionSearch extends Command {
+    private static CommonUi printer = new CommonUi();
+
     @Override
     public void execute(String[] input, ArrayList<Exercise> exercise,
```
Comment (tienkhoa16) [inline src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java]: Should the name of array type variable in plural form (i.e. inputs, exercises)?

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java [hunk 2]
```diff
@@ -2,15 +2,34 @@
...
                         String filePath, Storage storage, boolean[] endWorkoutSession) {
```
Comment (tienkhoa16) [inline src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java]: Should the boolean array name sounds more boolean (i.e. hasEndedWorkoutSessions)?

File: src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java [hunk 3]
```diff
@@ -38,4 +46,20 @@ public static void printHelp() {
                 "Go back to the workout menu."));
         printer.showToUser(helpMessage.toString().trim());
     }
+
+    public static void emptyInputError() {
+        System.out.println("Please enter something.");
+    }
+
+    public static void inputPrompt() {
+        System.out.print("Workout Session >>>>> ");
```
Comment (tienkhoa16) [inline src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java]: Workout Session is under Workout Menu, so is it clearer if we show "Workout Menu > Workout Session >>>>>" ?

**Review by tienkhoa16 — COMMENTED** [review]
LGTM! Just a few comments regarding variable naming and UI.

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java (hunk 1 above)
Comment (yujinyang1998) [reply src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java]: good call

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java (hunk 2 above)
Comment (yujinyang1998) [reply src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java]: good call

File: src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java (hunk 3 above)
Comment (yujinyang1998) [reply src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java]: great idea

File: src/main/java/workout/workoutsession/WorkoutSession.java [hunk 4]
```diff
@@ -52,10 +52,10 @@ public void workoutSessionStart() {
 
         while (!endWorkoutSession[0]) {
             try {
-                System.out.print("Workout Session >>>>> ");
+                WorkoutSessionUi.inputPrompt();
```
Comment (wgzesg) [inline src/main/java/workout/workoutsession/WorkoutSession.java:55]: Can create a new instance of workoutSessionUi and make the method non static. That would be more oop and better design

File: src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java [hunk 5]
```diff
@@ -38,4 +46,20 @@ public static void printHelp() {
                 "Go back to the workout menu."));
         printer.showToUser(helpMessage.toString().trim());
     }
+
+    public static void emptyInputError() {
+        System.out.println("Please enter something.");
+    }
+
```
Comment (wgzesg) [inline src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java:53]: Can try convert to throw Schwagserexceptions which can be handled by SchwasadsfgerHandler. That will help format the output which aligns with Ui output format

**Review by wgzesg — COMMENTED** [review]
Looks good. Ui can be consistent. Then let UI class handle all the

**Pull request merged**